__tests__/commands/changelogs/index.test.ts

```diff
@@ -21,7 +21,13 @@ describe('rdme changelogs', () => {
 
   beforeAll(() => {
     nock.disableNetConnect();
-    run = runCommand(Command);
+    run = (args: string[]) =>
+      runCommand(Command)(args).then(({ error, result }) => {
+        if (error) {
+          throw error;
+        }
+        return result;
+      });
```

Comment:
since these few lines of code are being reused in every test could they be made more generic and moved into `helpers/oclif`?

Reply:
https://github.com/readmeio/rdme/pull/1084/commits/f79579383a37487224e0aa6d71b2c91a6f8c097d